src/MultiField/MultiAssemblers.jl

```diff
@@ -70,34 +73,70 @@ end
 """
 MultiAssembler that produces SparseMatrices from the SparseArrays package
 """
-struct MultiSparseMatrixAssembler{E} <: MultiAssembler{SparseMatrixCSC{E,Int},Vector{E}}
+struct MultiSparseMatrixAssembler{E,M} <: MultiAssembler{AbstractSparseMatrix{E,Int},Vector{E}}
```

Comment:
```
struct MultiSparseMatrixAssembler{E,M} <: MultiAssembler{M,Vector{E}}
```
is enough. The inner constructor takes care of the rest.

Reply:
solved in 53d940e